web/views.py

```
@@ -763,6 +770,66 @@ def enroll_course(request, course_slug):
         return redirect("course_detail", slug=course_slug)
 
 
+@login_required
+def register_upcoming_sessions(request, slug):
```

Ruff (0.8.2)

Missing return type annotation for public function `register_upcoming_sessions`

(ANN201)